docs/installing-northstar/troubleshooting.md

```diff
@@ -208,6 +208,13 @@ If that solution doesn't work, you can try logging out of the EA App, closing al
 
 This error can also appear if you are pirating the game, which we neither condone nor support as stated [here](../faq.md#q-can-i-use-a-pirated-cracked-copy-titanfall-2-to-run-northstar)
 
+## Disable EA App overlay <a href="#ea-overlay" id="ea-overlay"></a>
+
+The purpose of disabling the EA App overlay is to increase performance, as it can be very taxing even on high end hardware. You can disable it by clicking on your profile in the top right of the EA App, clicking on the drop down arrow, going to settings, going to `Application`, then scrolling down and disabling the EA App overlay. This will disable invites, however [Northstar does not use these](../faq.md#q-im-trying-to-invite-a-friend-via-steam-origin-but-they-cannot-join-me)
```

Comment:
Wording

```suggestion
The purpose of disabling the EA App overlay is to increase performance, as it can be very taxing even on high end hardware. You can disable it by clicking on your profile in the top right of the EA App, clicking on the drop down arrow, going to settings, going to `Application`, then scrolling down and disabling the EA App overlay.
This will also disable invites, however [Northstar does not use these](../faq.md#q-im-trying-to-invite-a-friend-via-steam-origin-but-they-cannot-join-me)
```